tests/utils/TestHelper.ts

```diff
@@ -305,6 +308,34 @@ function assertFormDataMatchesObject(formData: FormData, obj: Report) {
     ).toEqual(expect.objectContaining(obj));
 }
 
+async function navigateToSidebarOption(index: number): Promise<void> {
+    const hintText = Localize.translateLocal('accessibilityHints.navigatesToChat');
+    const optionRows = screen.queryAllByAccessibilityHint(hintText);
+    fireEvent(optionRows[index], 'press');
+    await waitForBatchedUpdatesWithAct();
+}
+
+function beforeAllSetupUITests(shouldConnectToPusher = false) {
```

Comment:
This is no longer needed. There is `setupApp` and `setupGlobalFetchMock` that does the same